src/Compilers/VisualBasic/Portable/Symbols/AssemblySymbol.vb

```diff
@@ -341,16 +341,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Friend ReadOnly Property RuntimeSupportsDefaultInterfaceImplementation As Boolean
             Get
-                Return RuntimeSupportsFeature(SpecialMember.System_Runtime_CompilerServices_RuntimeFeature__DefaultImplementationsOfInterfaces)
+                Return SupportsRuntimeCapability(RuntimeCapability.DefaultImplementationsOfInterfaces)
             End Get
         End Property
 
-        Private Function RuntimeSupportsFeature(feature As SpecialMember) As Boolean
-            Debug.Assert(SpecialMembers.GetDescriptor(feature).DeclaringTypeId = SpecialType.System_Runtime_CompilerServices_RuntimeFeature)
+        Private Function IsStaticClass(namedTypeInternal As INamedTypeSymbolInternal) As Boolean Implements IAssemblySymbolInternal.IsStaticClass
+            Dim namedType = DirectCast(namedTypeInternal, NamedTypeSymbol)
+            Return namedType.IsClassType() AndAlso namedType.IsMetadataAbstract AndAlso namedType.IsMetadataSealed
```

Comment:
>namedType.IsMetadataAbstract [](http://example.com/codeflow?start=51&length=28)

`IsMetadataAbstract` does `Me.IsMustInherit OrElse Me.IsInterface`. We could inline the `IsMustInherit` logic here and drop the `IsInterface` (redundant since we're only interested in classes here).